utility/utils.py

```diff
@@ -146,6 +146,110 @@ def fuse_mount(fuse_clients, mounting_dir):
         log.error(e)
 
 
+def test_user_stats_consistency(primary_rgw_node, secondary_rgw_node):
+    """
+    verify and monitor sync consistency via bucket stats across sites.
+    """
+    log.info("Test number of users are consistent across sites")
+    total_users_on_primary = json.loads(
+        primary_rgw_node.exec_command(cmd="sudo radosgw-admin user list | wc -l")[0]
+    )
+    total_users_on_secondary = json.loads(
+        secondary_rgw_node.exec_command(cmd="sudo radosgw-admin user list | wc -l")[0]
+    )
+    user_list_primary = json.loads(
+        primary_rgw_node.exec_command(cmd="sudo radosgw-admin user list")[0]
+    )
+    if total_users_on_primary == total_users_on_secondary:
+        total_user = total_users_on_secondary - 2
```

Comment:
@viduship I did not follow this line , total_users_on_secondary - 2 ?

Reply:
@TejasC88 , 'radosgw-admin user list **| wc -l'**  will print 2 extra entries than the actual number